content/terraform/v1.11.x/docs/language/resources/ephemeral/index.mdx

```diff
@@ -5,39 +5,17 @@ description: Learn how to keep sensitive resource data out of state and plan fil
 
 # Ephemerality in resources
 
-Managing infrastructure often requires creating and handling sensitive values that you may not want Terraform to persist outside of the current operation. Terraform provides two tools for resources to manage data you do not want to store in state or plan files: the `ephemeral` resource block and ephemeral write-only arguments on specific resources.
+Managing infrastructure often requires creating and handling sensitive values that you may not want Terraform to persist outside of the current operation. Terraform provides two tools for resources to manage data you do not want to store in state or plan files: the `ephemeral` block and ephemeral write-only arguments on specific resources.
 
 ## Ephemeral resources
 
-Ephemeral resources are Terraform resources that are essentially temporary. Ephemeral resources have a unique lifecycle, and Terraform does not store information about ephemeral resources in state or plan files. Each `ephemeral` block describes one or more ephemeral resources, such as a temporary password or connection to another system.
+The `ephemeral` block defines resources that are essentially temporary. Ephemeral resources have a unique lifecycle, and Terraform does not store information about ephemeral resources in state or plan files. Each `ephemeral` block describes one or more ephemeral resources, such as a temporary password or connection to another system.
 
-In your configuration, you can only reference an `ephemeral` block in [other ephemeral contexts](/terraform/language/resources/ephemeral/reference#reference-ephemeral-resources).
-
-### Lifecycle
```

Comment:
Removed this context from here and added it to the ephemeral block reference page itself. I know it's more conceptual. but until we write more usage-focused page, I felt we should keep the info together on one page (even if it is the reference page)